blocks/editable/style.scss

```diff
@@ -47,6 +47,18 @@
 		background: $light-gray-400;
 	}
 
+	&:focus {
+		b, i, strong, em, del {
```

Comment:
I believe that ins was wanted as well

Reply:
@androb support for ins added, I also added support for sub and sup.